### DISCUSSION TITLE
Fehlermeldung: Lage außerhalb der Karte.

### DISCUSSION THREAD

**Discussion by Boomer4722** [devemux86/cruiser#426] [Q&A] [answered]
Wenn ich zb die Karte von Italien lade (egal ob Freizeitkarte oder OpenAndroMaps) sowie die Navigationsdaten von brouter oder graphhopper, so kann ich zwar Start und Ziel definieren, die Route wird auch berechnet und angezeigt, aber sobald ich auf das Navigationsicon gehe, kommt "Lage außerhalb der Karte".

Bug oder mache ich was falsch?

**Comment by devemux86** [comment, marked as answer]
@Boomer4722 

This message indicates that the GPS location is outside the loaded offline map.
Where is your GPS location? Outside of Italy?

Diese Meldung zeigt an, dass der GPS-Standort außerhalb der geladenen Offline-Karte liegt.
Wo befindet sich Ihr GPS-Standort? Außerhalb von Italien?

**Reply by 0709wiwiwi** [reply]
Or load the world map...Should be standard.

**Comment by devemux86** [comment]
@Boomer4722 

If your location is outside the loaded offline map, you can also simulate navigation by long pressing the navigation button.

Wenn sich Ihr Standort außerhalb der geladenen Offline-Karte befindet, können Sie die Navigation auch durch langes Drücken der Navigationstaste simulieren.

**Reply by 0709wiwiwi** [reply]
Or load the world map...Should be standard.

**Comment by Boomer4722** [comment]
Die Antwort war sehr (!) hilfreich! Danke! 

Es lag genau daran, dass meine tatsächliche Position außerhalb der Karte lag. Darauf hätte ich auch selbst kommen können. :-(

Was ist mit der Weltkarte gemeint, wo finde ich die und warum würde die das Problem auch lösen?

**Reply by devemux86** [reply]
@Boomer4722 

A small world map is available in Mapsforge maps, which can be used together with the other detailed maps.
Please see the image.

In Mapsforge maps ist eine kleine Weltkarte verfügbar, die zusammen mit den anderen detaillierten Karten verwendet werden kann.
Bitte sehen Sie sich das Bild an.

<img src="https://github.com/devemux86/cruiser/assets/3484020/e88d9e28-22fd-404c-91fd-6baf38973277" width="50%"/>

**Comment by Boomer4722** [comment]
Danke für die schnelle und hilfreiche Antworten.😍

Übrigens eine tolle App, die ich selbstverständlich abonniert habe.

Was ich mit allerdings wünschen würde, wäre eine schnellere Umschaltmöglichkeit zwischen verschiedenen Karten sowie GraphHopper Daten. Ich finde es etwas umständlich, jedesmal über Downloads zu gehen. Oder gibt es eine einfachere Möglichkeit und ich war schon wieder blind? Bei dem Wechsel zwischen verschiedenen Themes würde ich mir ebenfalls eine schnellere Option wünschen. 

Liebe Grüße

**Reply by devemux86** [reply]
@Boomer4722 

Thank you for your kind words!
If you have any questions or ideas for improvements and new features, please let us know.

The themes can be changed in the Map / Theme menu.
The maps in this version are changed in downloads, next version has a global offline map menu.

---

Vielen Dank für die freundlichen Worte!
Wenn Sie Fragen oder Ideen für Verbesserungen und neue Funktionen haben, lassen Sie es uns bitte wissen.

Die Themen können im Menü Karte / Thema geändert werden.
Die Karten in dieser Version sind in Downloads geändert, nächste Version hat eine globale Offline-Karten-Menü.

**Comment by Boomer4722** [comment]
Die Umschaltmöglichkeit für die Themen hatte ich schon gefunden, würde mir aber einen schneller Zugriff wünschen.

Auf ein erweitertes Offline-Karten-Menü freue ich mich schon. Die App wird immer besser...

Wäre es nicht auch möglich, die GraphHopper Daten automatisch entsprechend der aktuelle gewählten Karte auszuwählen? Also ohne sie explizit zu aktivieren!?

**Reply by devemux86** [reply]
@Boomer4722 

> Wäre es nicht auch möglich, die GraphHopper Daten automatisch entsprechend der aktuelle gewählten Karte auszuwählen? Also ohne sie explizit zu aktivieren!?

This is more complicated as GraphHopper only allows 1 country to be activated at a time.
So if we activate 2 maps, the user has to choose which GraphHopper data to activate.
We recommend to check BRouter (offline) routing, it allows routing over countries. :slightly_smiling_face: 

Dies ist komplizierter, da bei GraphHopper jeweils nur 1 Land aktiviert werden kann.
Wenn wir also 2 Karten aktivieren, muss der Benutzer auswählen, welche GraphHopper-Daten aktiviert werden sollen.
Wir empfehlen, BRouter (offline) Routing zu prüfen, es erlaubt Routing über Länder. :slightly_smiling_face:

**Comment by Boomer4722** [comment]
Ich hatte brouter schon bei mir bekannten Strecken (bike) getestet. Dies vorgeschlagenen Routen würde ein Ortskundiger niemals fahren.

Bei GraphHopper war dies (in meinem Fall) ganz anders: Es wurde exakt (!) die Route vorgeschlagen, die ich als Ortkundiger auch fahre. Dies hat mich überzeugt!

Bei brouter ist mir im Zusammenhang mit Cruiser aufgefallen, dass bei ganz unterschiedlichen Cruiser-Profilen, die gleichen brouter-Profile gewählt werden. Bei "zu Fuß", "wandern" und "laufen" wird jeweils "hiking-mountain" verwendet. Dies macht meiner Meinung nach wenig Sinn und es wäre wohl sinnvoller zu differenzieren. 

Brouter bietet ja fast 30 Profile. Vielleicht wäre es auch sinnvoll, wenn man diese in der App frei dem jeweiligen Cruiser Profil zuordnen könnte.

**Reply by devemux86** [reply]
@Boomer4722 

BRouter has one pedestrian profile, but it has many options to customize it.
You can open external BRouter profiles in Cruiser's advanced BRouter settings.

BRouter hat ein einziges Fußgänger Profil, aber es gibt viele Optionen, um es anzupassen.
Sie können externe BRouter-Profile in den erweiterten BRouter-Einstellungen von Cruiser öffnen.

**Comment by devemux86** [comment]
@Boomer4722 

Can you write some examples of pedestrian routes so that we can check them?

Können Sie einige Beispiele für Fußgängerrouten aufschreiben, damit wir sie überprüfen können?
